src/main/java/bio/terra/service/profile/flight/ProfileMapKeys.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.service.profile.flight;
+
+public final class ProfileMapKeys {
+    public static final String PROFILE_ID = "profileId";
```

Comment:
Would it make sense to just have these as part of the main flight class?

Reply:
I'm not sure what you mean "the main flight class". I expect that parameter will be used in several flights.

We do not have a good story for the organization of the keys. It might be better to have a single file with all of them, so we could share everything and not risk a string collision in the hash table. Typically, the names are prefixed by the service - but not consistently.

When I write a general purpose step - AuthorizeBillingProfileUseStep in this case - I will have the associated keys in a file. So PROFILE_MODEL in ProfileMapKeys. And then the profile steps use PROFILE_ID, so it goes in the same place.

We have some common key names in JobMapKeys that all flights use.

Reply:
Sorry, I meant the flight in this package that uses these keys and instantiates the steps that may also use them

Reply:
I added an item in the Design Reviews document that describes the key naming issue and suggests to alternatives. I won't change this current code.